web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -408,83 +405,6 @@ public static boolean isModulePackageNameInTaskClass(String modulePackageName, S
 	}
 	
 	/**
-	 * Method visibility is package-private for testing
-	 *
-	 * @param mod
-	 * @param realPath
-	 * @should prefix messages with module id
-	 * @should not prefix messages with module id if override setting is specified
-	 */
-	static void copyModuleMessagesIntoWebapp(Module mod, String realPath) {
-		for (Entry<String, Properties> localeEntry : mod.getMessages().entrySet()) {
-			if (log.isDebugEnabled()) {
-				log.debug("Copying message property file: " + localeEntry.getKey());
-			}
-			
-			Properties props = localeEntry.getValue();
-			
-			if (!"true".equalsIgnoreCase(props
-			        .getProperty(ModuleConstants.MESSAGE_PROPERTY_ALLOW_KEYS_OUTSIDE_OF_MODULE_NAMESPACE))) {
```

Comment:
Great, @tomaszmueller, please be sure to test the reference application with your changes. The easiest way to do it is to apply your fix to the 1.11.x branch and run it with the set of 2.x modules which you can get from our CI, e.g. https://ci.openmrs.org/browse/REFAPP-OMODDISTRO-3320/artifact/shared/openmrs-2.x-modules-zip/referenceapplication-package-2.0.3320-distro.zip